### PR TITLE
Create a proper readme

### PR DESCRIPTION
**Pull request by xeome**
Create a readme with information about dependencies, installation, features.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,32 @@
+
 # pardus-image-writer
 
-a Python GTK App to burn ISO files on USB Flash sticks.
\ No newline at end of file
+A tool for writing image files to removable storage devices such as USB drives and SD cards.
+
+# Introduction
+
+pardus-image-writer is a simple and user-friendly tool for writing image files to removable storage devices. It's designed to be fast, reliable, and easy to use, making it the perfect solution for anyone who needs to write images to removable storage quickly and efficiently.
```
Comment (eminfedar) [inline README.md]: I think the summary at Line 4 is sufficient. You can remove this.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,32 @@
...
+
+# Features
+- Cross-platform compatibility: pardus-image-writer is written in Python, works on many linux distributions.
```
Comment (eminfedar) [inline README.md]: Cross Platform compatibility generally means: "it works Windows, Linux & macOS"

This is "Linux" only.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,32 @@
...
+
+To use pardus-image-writer, you can use the following command to install necessary dependencies:
+
+```bash
+pip install setuptools pyudev
+```
+
+# Installation
+
+To install pardus-image-writer, follow these steps:
```
Comment (eminfedar) [inline README.md:19]: Installing from .deb files is easy to handle for end users.

These steps can be written as a second alternative.

Please add these steps above:
```suggestion
1. Download the latest version of .deb file from [Releases](https://github.com/pardus/pardus-image-writer/releases) page.
2. Install the .deb file with Pardus Package Installer by double clicking on it. (or use `sudo apt install pardus-image-writer_0.4.0.Beta1_all.deb`)
```

**Review by eminfedar — CHANGES_REQUESTED** [review]
I requested some changes. After the changes, this PR is good to me.

**Comment by xeome** [comment]
Hello, thanks for the suggestions, I have made the changes that you requested with minor additions.

**Comment by fthaltun** [comment]
We were thinking of using a standard readme template in all pardus projects. We were going to implement this in the near future; If you want, you can send a request by following the examples below.

For example;
https://github.com/pardus/pardus-mycomputer/blob/master/README.md
https://github.com/pardus/pardus-software/blob/master/README.md

**Pull request closed without merging**